### ISSUE TITLE
[A11y] Work Show Page-All - Institution Image

### ISSUE THREAD

**Issue by jechols** [OregonDigital/OD2#2013]
### Operating System

Windows

### Browser

Edge

### Affected page

Work Show

### Admin user?

Yes

### Issue Description

https://staging.oregondigital.org/concern/generics/df727084j
https://staging.oregondigital.org/concern/documents/df727057n
https://staging.oregondigital.org/concern/images/1g05fb620

The "University of Oregon | Libraries" image is coded as meaningful, and has the `alt` attribute "University of Oregon". One or both of these are incorrect.

### Suggested Fix

By its placement on the page, one could potentially assume it's designating the institution responsible for the page's content. If that's the purpose of this image, its alt attribute should have that information, though it could be argued the lack of explanation visually is its own problem.

If the image is meant to convey nothing other than the text in the image, the `alt` attribute must contain all the text in that image, not just part of the text.

If the image can be argued as being purely a decorative image, its `alt` attribute should be null (`alt=""`). I don't think this is the case, but it's very subjective. I don't know that I really understand the purpose of this image.

### Can anybody QA or does this need an expert?

A11y familiarity recommended

### QA Steps

Verify the alt attribute makes sense in context. Which is super subjective to decide, but at the very least if we're just trying to reproduce the image's text, we need to do so faithfully.
